src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+const PRICE_FOR_DAY = 40;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
```

Comment:
The variable `res` should be declared as a `const` since you do not reassign it; instead, you're only modifying its value. This helps to prevent accidental reassignment and makes the intention of the code clearer.